custom/eqa/expressions.py

```diff
@@ -16,7 +16,13 @@
 
 
 def get_val(form, path, default=0):
-    return int(form.get_data(path)) if form else default
+    if not form:
+        return default
+    question_value = form.get_data(path)
+    try:
+        return int(question_value)
+    except ValueError:
```

Comment:
Ah that's good to know, `question_value` could be `None`, and I would want `default` in that case. I'll update it.